Platform/RaptorlakeBoardPkg/BoardConfigRplp.py

```diff
@@ -0,0 +1,703 @@
+            self.PAYLOAD_SIZE         = 0x00024000
+
+        if self.ENABLE_SOURCE_DEBUG:
+            self.STAGE1B_SIZE += 0x4000
+            if self.SKIP_STAGE1A_SOURCE_DEBUG == 0:
+                self.STAGE1A_SIZE += 0x4000
+
+
+        self.UEFI_VARIABLE_SIZE = 0x1000
+        if len(self._PAYLOAD_NAME.split(';')) > 1:
+            self.UEFI_VARIABLE_SIZE = 0x00040000
+        self.UCODE_SIZE           = 0x000B7000
+        self.MRCDATA_SIZE         = 0x00010000
+        self.CFGDATA_SIZE         = 0x00004000
+        self.KEYHASH_SIZE         = 0x00001000
+        self.VARIABLE_SIZE        = 0x00002000
+        self.SBLRSVD_SIZE         = 0x00001000
+        self.FWUPDATE_SIZE        = 0x00020000 if self.ENABLE_FWU else 0
+
+        self.SIIPFW_SIZE = 0x1000
+
+        # Declaring UCODE_SLOT_SIZE populates the uCode region with as many
+        # uCode slots as possible, even if there aren't enough uCode patches
+        # to fill each slot. This can have a slight performance impact on SBL,
+        # as it tries to load dummy uCode patches.
+        self.UCODE_SLOT_SIZE      = 0x4C000
+
+        # If ENABLE_SBL_RESILIENCY is 1, BiosRedAssistance FIT strap setting
+        # needs to be manually changed to Enabled and TopSwapOverride
+        # flash setting needs to be manually changed to 4MB in stitch config
+        self.ENABLE_SBL_RESILIENCY = 0
+        self.BUILD_IDENTICAL_TS    = 0
+
+        if self.ENABLE_SBL_RESILIENCY:
+            self.BUILD_IDENTICAL_TS    = 1
+
+        if self.BUILD_IDENTICAL_TS:
+            self.TOP_SWAP_SIZE         = 0x400000
+            self.REDUNDANT_SIZE        = self.STAGE2_SIZE + self.FWUPDATE_SIZE + self.CFGDATA_SIZE + \
+                                         self.KEYHASH_SIZE
+        else:
+            self.TOP_SWAP_SIZE        = 0x80000
+            self.REDUNDANT_SIZE       = self.UCODE_SIZE + self.STAGE2_SIZE + self.STAGE1B_SIZE + \
+                                        self.FWUPDATE_SIZE + self.CFGDATA_SIZE + self.KEYHASH_SIZE
+
+        self.FUSA_SUPPORT         = 0
+        self.FUSA_SIZE            = 0
+
+        # These can be overridden by FUSA .dlt file.
+        self.ENABLE_TCC           = 0
+        self.ENABLE_TSN           = 0
+
+        if self.FUSA_SUPPORT:
+            self.ENABLE_PRE_OS_CHECKER = 0
+            if self.ENABLE_PRE_OS_CHECKER:
+                self.POSC_SIZE = 0x00080000
+                self.SIIPFW_SIZE += self.POSC_SIZE
+            self.FUSA_BIST_PATTERN_SUPPORT  = 0
+            if self.FUSA_BIST_PATTERN_SUPPORT:
+                self.FUSA_PATTERN_IS_FFS        = 0
+                self.FUSA_STARTUP_BIST_FILE     = ''
+                self._FUSA_PERIODIC_BIST_FILE    = ''
+                self.FSBP_SIZE        = 0x1000000
+                self.FPBP_SIZE        = 0x1000000
+                self.FUSA_SIZE        = 0x1000 + self.FPBP_SIZE + self.FSBP_SIZE
+
+            # Allow TCC to be enabled by FuSa dlt file
```

Comment:
Suggest removing these codes (#218~#237), since these codes are trying to modify the values of ENABLE_TCC and ENABLE_TSN according to the settings in CfgData dlt files.
It resulted in complicated logic of TCC and TSN enablement, especially when ENABLE_TCC / ENABLE_TSN are used during build time, while the TccEnable / PchTsnEnable in DLT files can be used during stitch and run time

Reply:
Explained via email. This is removed now.